### ISSUE TITLE
question about the recurrent

### ISSUE THREAD

**Issue by rainbow979** [ikostrikov/pytorch-a2c-ppo-acktr#289]
I have a question about the recurrent.
Under the recurrent setting, there is not the computation graph betweent step `t` and step `t-1`, how to backprop gradient?
https://github.com/ikostrikov/pytorch-a2c-ppo-acktr-gail/blob/master/a2c_ppo_acktr/storage.py#L145

**Comment by rainbow979** [comment]
I got the reply from reddit and did solve my question.
https://www.reddit.com/r/reinforcementlearning/comments/t2oi2g/comment/hyngt09/?utm_source=share&utm_medium=web2x&context=3
